config/routes.rb

```diff
@@ -8,5 +8,5 @@
   get '/signin' => 'sessions#new', :as => :signin
   get '/signout' => 'sessions#destroy', :as => :signout
 
-  root to: 'pages#home'
+  root to: 'github_events#index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.